python-lib/dku_io_utils/recipe_config_loading.py

```diff
@@ -179,6 +179,11 @@ def load_predict_config():
     params["quantiles"] = convert_confidence_interval_to_quantiles(params["confidence_interval"])
     params["include_history"] = recipe_config.get("include_history", False)
 
+    params["sampling_method"] = recipe_config.get("sampling_method", "last_records")
+    params["history_length_limit"] = None
+    if params["sampling_method"] == "last_records":
+        params["history_length_limit"] = recipe_config.get("number_records", 1000)
```

Comment:
We should add parameter validation here, to guarantee that this number is defined and > 0

Reply:
done in 5d38dcf